### ISSUE TITLE
Commas in function that passed to `@macro` argument are treating as actual commas.

### ISSUE THREAD

**Issue by defaultzone** [ClueLang/Clue#142]
```lua
@macro call(func) { ($func)() }

$call!(fn {
    local a, b = "", false
})
```
```bash
$ clue test.clue 
Error in test.clue:3:1!
Error: "Too many arguments passed to macro"
```

**Comment by Maiori44** [comment]
This has already been fixed in 4.0, but I can bring it to 3.0 if/when a new patch happens.
Until then, you could fix the issue by slightly changing the code:
```lua
@macro call(func) { $func() }

$call!((fn {
    local a, b = "", false
}))
```

**Comment by Maiori44** [comment]
The fix has been brought back to 3.0 and will be included in the next patch.
